frontend/containers/profile-header/component.tsx

```diff
@@ -51,12 +51,12 @@ export const ProfileHeader: FC<ProfileHeaderProps> = ({
 
   return (
     <div className={className}>
```

Comment:
There are multiple layout issues with this component. I'm adding here a few screenshots.

<table>
<tr>
<th>Screenshot
<th>Problem
<tr>
<td width="300">
<img src="https://user-images.githubusercontent.com/6073968/195854359-98afa20b-fb66-4b94-9f35-fbb491d4d16d.png" alt="The favorite button is left aligned and the contact button takes the full width" height="400"  />
<td>On an iPhone 5, the buttons are not displayed consistently. I don't think both can fit side-by-side at 320px.
<tr>
<td width="300">
<img src="https://user-images.githubusercontent.com/6073968/195854350-3789fc73-d9f9-473d-8511-bdf8d706b5d7.png" alt="At an intermediate screen size, the contact button grows in size disproportionately" width="300"  />
<td>At an intermediate screen size, the “Contact” button grows in size disproportionately.
<tr>
<td width="300">
<img src="https://user-images.githubusercontent.com/6073968/195854326-c064193e-8a63-49b9-bace-650ad866bbe7.png" alt="The investor's logo is stretched" width="300"  />
<td>
The logo can also stretch and lose its square ratio. We can maybe change at which breakpoint we switch from the mobile to the desktop layout.
</table>